### ISSUE TITLE
Failing tests

### ISSUE THREAD

**Issue by lethal-guitar** [neuviemeporte/mzretools#1]
I've just built the current `master` branch, and noticed that some tests are failing:

```
[  FAILED  ] 4 tests, listed below:
[  FAILED  ] Dos.MzHeader
[  FAILED  ] AnalysisTest.FindRoutines
[  FAILED  ] AnalysisTest.FindFarRoutines
[  FAILED  ] AnalysisTest.CodeCompare
```

Is this to be expected, or am I missing something?

**Comment by neuviemeporte** [comment]
Hi Nikolai, thanks for letting me know. No, it was not supposed to fail, but was failing because it could not find the path for the sample Hello World MZ application used in the test. I made a symlink to it in my environment a long time ago and did not bother to check if a freshly cloned repo builds in a long time.

I pushed out a commit changing the path in the tests to address the parent directory as a quick-and-dirty fix.
